sympy/sets/sets.py

```diff
@@ -1845,8 +1845,15 @@ class FiniteSet(Set, EvalfMixin):
     True
 
     >>> members = [1, 2, 3, 4]
-    >>> FiniteSet(*members)
+    >>> f = FiniteSet(*members)
```

Comment:
@smichr yeah this looks more simplified and appropriate.
I will make the necessary changes.
Thanks!!